awx/ui_next/src/components/MultiButtonToggle/MultiButtonToggle.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+import styled from 'styled-components';
+import { Button } from '@patternfly/react-core';
+import ButtonGroup from './ButtonGroup';
+
+const SmallButton = styled(Button)`
+  padding: 3px 8px;
+  font-size: var(--pf-global--FontSize--xs);
+`;
+
+function MultiButtonToggle({ buttons, currentValue, onChange }) {
+  const setValue = newValue => {
+    if (currentValue !== newValue) {
+      onChange(newValue);
+    }
+  };
+
+  return (
+    <ButtonGroup>
+      {buttons.map(([value, label]) => (
+        <SmallButton
+          key={label}
+          onClick={() => setValue(value)}
+          variant={currentValue === value ? 'primary' : 'secondary'}
+        >
+          {label}
+        </SmallButton>
+      ))}
+    </ButtonGroup>
+  );
+}
+
+const buttonsPropType = {
```

Comment:
Nice. I'm actually surprised prop-types doesn't already have a tuple type or something that can do this for is, but I couldn't find it either.